subsys/net/lib/lwm2m/Kconfig

```diff
@@ -23,6 +23,20 @@ config LWM2M_DTLS_SUPPORT
 	select NET_SOCKETS_SOCKOPT_TLS
 	select NET_SOCKETS_ENABLE_DTLS
 
+choice
+	prompt "LwM2M Server object version"
+	default LWM2M_SERVER_OBJECT_VERSION_1_0
+	help
+	  Select which version of the Server object should be used.
+
+config LWM2M_SERVER_OBJECT_VERSION_1_0
+	bool "Server object version 1.0"
+
+config LWM2M_SERVER_OBJECT_VERSION_1_1
+	bool "Server object version 1.1 [EXPERIMENTAL]"
```

Comment:
You should add `select EXPERIMENTAL` so that it will generate a warning during build time.

Reply:
Ah OK I need to check